conf/environment.conf

```diff
@@ -11,6 +11,8 @@
 # Allowed settings and default values:
 
 # modulepath = ./modules:$basemodulepath
-# manifest = (default_manifest from puppet.conf; defaults to ./manifests)
+# manifest = (default_manifest from puppet.conf, which defaults to ./manifests)
 # config_version = (no script; Puppet will use the time the catalog was compiled)
-# environment_timeout = (environment_timeout from puppet.conf; defaults to unlimited)
+# environment_timeout = (environment_timeout from puppet.conf, which defaults to 0)
+    # Note: unless you're doing something unusual, we recommend only setting
```

Comment:
How about "unless you know you need it" ...?


![](http://img.pandawhale.com/94396-I-myself-am-strange-and-unusua-41rA.gif)